examples/example_equilibrate.py

```diff
@@ -0,0 +1,373 @@
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+# Copyright (C) 2012 - 2015 by the BurnMan team, released under the GNU
+# GPL v2 or later.
+
+'''
+example_equilibrate
+--------------------
+
+This example demonstrates how burnman may be used to calculate the
+equilibrium phase proportions and compositions for an assemblage
+of a fixed bulk composition.
+
+*Uses:*
+
+* :doc:`mineral_database`
+* :class:`burnman.composite.Composite`
+* :func:`burnman.equilibrate.equilibrate`
+'''
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+import sys
+# hack to allow scripts to be placed in subdirectories next to burnman:
+if not os.path.exists('burnman') and os.path.exists('../burnman'):
+    sys.path.insert(1, os.path.abspath('..'))
+
+import numpy as np
+import matplotlib.pyplot as plt
+plt.style.use('ggplot')
+
+import burnman
+from burnman.minerals import HP_2011_ds62, SLB_2011, JH_2015
+from burnman import equilibrate
+
+ordering = False # This example plots the state of order of the Jennings and Holland orthopyroxene in the simple en-fs binary at 1 bar.
+aluminosilicates = False # The example plots the andalusite-sillimanite-kyanite phase diagram
+gt_solvus = False # This example demonstrates the shape of the pyrope-grossular solvus
+lower_mantle = True # This example calculates temperatures and assemblage properties along an isentrope in the lower mantle
+upper_mantle = False # This example produces a 2D grid of the ol-opx-gt field
+olivine_polymorphs = True # DOESN'T SOLVE IF LOWER MANTLE OR UPPER MANTLE ARE RUN BEFOREHAND. NEED TO ROOT OUT THIS WEIRD BUG! This example produces a P-T pseudosection for a fo90 composition 
```

Comment:
Is this bug solved? At least it was no problem for me.

Reply:
Yes, I fixed this yesterday. Repeated function calls in python are peculiar. I added a note at the very top of equilibrate to remind me not to reverse the change in the future.